flashlight/app/lm/Trainer.h

```diff
@@ -29,6 +29,8 @@
 #include "flashlight/lib/text/tokenizer/PartialFileReader.h"
 #include "flashlight/lib/text/tokenizer/Tokenizer.h"
 
+using namespace fl::pkg::text;
```

Comment:
Can we remove this from the header/put it in the cpp file?

Reply:
Yes, good catch.